.github/workflows/golangci-lint.yml

```diff
@@ -20,20 +20,12 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
           version: v${{ steps.tool-versions.outputs.golangci-lint_version }}
-
-          # Optional: working directory, useful for monorepos
-          # working-directory: somedir
-
-          # Optional: golangci-lint command line arguments.
           args: --enable=gofmt --tests=false --exclude-use-default --timeout=5m0s
```

Comment:
This command won't produce a report, add `--out-format checkstyle:golangci-lint-report.xml` After done, verify in the corresponding workflow, that the artifact is uploading and in sonar-scan.yml, you get it (take a look [here](https://github.com/smartcontractkit/chainlink-solana/actions/runs/6143326053/job/16666756273#step:7:3), the path for linter should not be empty)